styles.css

```diff
@@ -0,0 +1,108 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Helvetica Neue',
+    Arial,
+    sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+h1 {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+
+#title {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
```

Comment:
- [ ] Remove one of these selectors as they behave the same and are considered repeated styles.

Reply:
It would be better to explain your comments politely. For example, you can use this sentence at the beginning of your comment "I think it would be better to use semantic tags" 👎